src/foam/u2/Element.js

```diff
@@ -2426,6 +2430,25 @@ foam.CLASS({
         // a promise will generate an intermediate null value.
         return arr[0].restrictDisplayMode(arr[1] || DisplayMode.HIDDEN)
       });
+    },
+
+    function createPermissionFor(x, cls) {
```

Comment:
How is this different than createVisibilityFor()?

Reply:
@KernelDeimos, correct me if I'm wrong. I think all we want is to bypass the combineControllerModeAndVisibility_ and  controllerMode$